### ISSUE TITLE
Feature request: ability to add SDR metadata-only items to an exhibit

### ISSUE THREAD

**Issue by caaster** [sul-dlss/exhibits#1932]
**Use Case**

The event coordinator (Sharyn Nantuna) for Stanford SITES (Stanford Institute for Theoretical Economics) reached out to DLSS in 2019/20 to discuss full text processing and SDR preservation for the born digital papers and metadata for presentations given at their annual SITES conference. They also needed to migrate their website dedicated to discovery of information 
about the annual SITES conferences alongside the papers presented there. They wanted more robust discovery for the papers (and more context) than SearchWorks affords, so they decided to investigate using Spotlight for this purpose.

Sharyn will start building a Spotlight exhibit in Sept 2020. There is one challenge she has in doing so. Of the 2,412 papers presented at the conference since 1989, 1,160 of these papers (https://argo.stanford.edu/catalogf%5Bnonhydrus_apo_title_ssim%5D%5B%5D=SITE+Archive+-+citation) have no digital version and exist in the SDR as metadata only. Sharyn would like to be able to add these metadata only items to her exhibit, so that they can be searchable and discoverable in the exhibit, in the same way that SDR items with files can be discovered (https://argo.stanford.edu/catalog?f%5Bnonhydrus_apo_title_ssim%5D%5B%5D=SITE+Archive+-+world). Browse categories will be created for all items added to the exhibit. Right now the plan is to create browse categories by year, and also by theme/topic.

All PDFs currently in the SDR are full-text search enabled and Sharyn is very pleased to have the search across feature in this exhibit. New content will be added to the SDR annually after each conference, and going forward all papers will be in digital format, processed for full text search by DPG. 

As a workaround for the 1,160 SDR items that are metadata only, Sharyn will provide lists of PURLs pointing to metadata only items in the exhibit, most likely in a list on feature pages. One feature page will be created for each annual conference. Ideally, Sharyn would like to be able to add these metadata only items to her exhibit.

**Comment by ggeisler** [comment]
@caaster I know this isn't the biggest concern of this use case, but I wanted to note that the proposed [browse grouping feature](https://github.com/projectblacklight/spotlight/issues/2562) seems like it would be useful for this exhibit. One group could be "Year" and would help the user interested in browsing by conference year filter out the theme/topic-related browse categories. Depending on how many there are, other groups could be the most prominent themes/topics.

**Comment by caaster** [comment]
@ggeisler I am still learning about this content (I don't train the creator until this Friday), but Sharyn mentioned that the themes cross year boundaries, too. I'll clarify here on this ticket if it makes sense to do so.

**Comment by ggeisler** [comment]
@caaster I don't think that matters. If the curator does in fact "... create browse categories by year, and also by theme/topic." then the browse categories that are year-based would be added to the "Year" browse grouping, and the browse categories that are created based on theme/topic could go into a "Theme" group.

Or, if there are a lot of theme/topic based browse categories, and some of the themes are somewhat related (e.g., three categories that fit into "Macroeconomics", five that fit into "Microeconomics", etc. those could be browse groups.

Of course, none of this is really relevant now -- since we don't have a browse grouping feature -- and I wasn't suggesting you needed to consider it when training or working with the curator, or need to create a ticket for it. Just pointing out that it sounds like this is another exhibit that probably could utilize the feature if we decide to invest the effort into creating it.

**Comment by caaster** [comment]
There was an acceptable workaround used for this exhibit. So I am closing this ticket. There are other more compelling use cases for this feature request; when/if it becomes a priority we should create a new ticket.
